.gitignore

```diff
@@ -4,4 +4,7 @@
 **/shapelibraries/*.lcsz
 **/*.credentials.local
 **/manifest.local.json
-**/data-connector/bin
\ No newline at end of file
+**/data-connector/bin
+
+# whitelist
+!icon-repository-dragndrop/package.zip
```

Comment:
It seems like committing the package.zip was intentional. What is the goal here? Can we use GitHub releases to do the same thing without mucking up the repo with binary blobs?

Reply:
Yeah I think that's a good idea